lib/miq_automation_engine/models/miq_ae_datastore.rb

```diff
@@ -165,15 +165,20 @@ def self.seed_default_namespace
   def self.reset_to_defaults
     raise "Datastore directory [#{DATASTORE_DIRECTORY}] not found" unless Dir.exist?(DATASTORE_DIRECTORY)
     saved_attrs = preserved_attrs_for_domains
-    Dir.glob(DATASTORE_DIRECTORY.join("*", MiqAeDomain::DOMAIN_YAML_FILENAME)).each do |domain_file|
-      domain_name = File.basename(File.dirname(domain_file))
+    default_domain_names.each do |domain_name|
       reset_domain(DATASTORE_DIRECTORY, domain_name, Tenant.root_tenant)
     end
 
     restore_attrs_for_domains(saved_attrs)
     reset_default_namespace
   end
 
+  def self.default_domain_names
```

Comment:
@pkomanek Please create a test for this new method.  Locally we should always get the ManageIQ domain back.